examples/gbasf2_fei_training/fei_grid_workflow.py

```diff
@@ -0,0 +1,473 @@
+from multiprocessing import Pool
+import os
+import glob
+import shlex
+import pickle
+import json
+import datetime
+import subprocess
+
+import b2luigi as luigi
+from b2luigi.basf2_helper.tasks import Basf2PathTask
+from b2luigi.basf2_helper.utils import get_basf2_git_hash
+from b2luigi.batch.processes.gbasf2 import run_with_gbasf2
+
+from B_generic_train import create_fei_path, get_particles
+import fei
+
+
+def shell_command(cmd):
+    os.system(cmd)
```

Comment:
Why ??? I don't see where this is used and I don't see what this adds besides aliasing a short command with a slightly longer command. Anyway, `os.system` is outdated and not recommended. If you _really_ think you need to run your command in a shell, use `subprocess.run(cmd, shell=True)` (`subprocess.call(cmd, shell=True)` also works, `run` is newer and the currently recommended subprocess function). But if you command doesn't need to run in a dedicated shell (bash/zsh/...), try running it without `shell=True`, e.g. just
``` python
cmd_list = shlex.split(cmd_str)
subprocess.run(comd_list, check=True)
```

Reply:
yeah... this was my solution to use it for multiprocessing. I'll redo that in a proper way. And here, I need to perform a shell command ton run the fei merging executable.

Reply:
I don't see where the function `shell_command` does multiprocessing, as far as I see it there is no advantage to using `os.system` directly, it's basically an alias that you created.

We already a [Basf2nTupleMergeTask](https://github.com/nils-braun/b2luigi/blob/main/b2luigi/basf2_helper/tasks.py#L115) which interally calls `fei_merge_files`/`analysis-fei-mergefiles` via`subprocess.check_call` (equivalent to the newer `subprocess.run(check=True`)  and this works perfectly well as a mere subprocess without requiring a shell with e.g. `shell=True`.

Reply:
The function `shell_command` does no multiprocessing, it is **used** for multiprocessing:

https://github.com/ArturAkh/b2luigi/blob/fei_with_gbasf2/examples/gbasf2_fei_training/fei_grid_workflow.py#L227-L228

Since I have very specific requirements for my merging building block, I'd like to not inherit from `Basf2nTupleMergeTask` if possible.

I would rather consider that as a future improvement to put it better together - after this pull request is merged.

Reply:
As said before, I'll the multi-processed merge command in a better way.

Reply:
By using `p.map` doesn require defining your own `shell_command` function, you can just use something like
``` python
 	        p = Pool(self.ncpus)
	        p.map(subprocess.check_all, cmds)
```
Experience python programmers should recognize `subprocess.check_call`.

Only issue is that subprocess.run requires a command as a list instead of as a string, but you can split strings into command lists via `shlex.split(cmd)`.

Reply:
As said before: I will rework the command for sure...

And yes, when it comes to proper python style, I'm not that experienced. I'm already satisfied, when thing work as I expect them to work.

Reply:
Ok, should be resolved now.